src/dos/drive_overlay.cpp

```diff
@@ -1088,7 +1088,9 @@ void Overlay_Drive::remove_special_file_from_disk(const char* dosname, const cha
 	safe_strcpy(overlayname, overlaydir);
 	safe_strcat(overlayname, name.c_str());
 	CROSS_FILENAME(overlayname);
-	if(unlink(overlayname) != 0) E_Exit("Failed removal of %s",overlayname);
+	if (!simple_delete_file(overlayname)) {
+		E_Exit("Failed removal of %s", overlayname);
```

Comment:
Hard-exiting if a file cannot be deleted is a bit rich... 😅 

Could you change it to `LOG_ERR` while you're at it?

Reply:
OK, changed. Also in one more place.